filebeat/module/mariadb/slowlog/_meta/fields.yml

```diff
@@ -0,0 +1,73 @@
+- name: slowlog
+  type: group
+  release: experimental
+  description: >
+    Contains fields from the MySQL slow logs.
+  fields:
+    - name: user
+      description: >
+        The MySQL user that created the query.
+    - name: current_user
+      description: >
+        The MySQL user that created the query.
+    - name: host
+      description: >
+        The host from where the user that created the query logged in.
+    - name: client_host
+      description: >
+        The host from where the user that created the query logged in.
+    - name: ip
+      description: >
+        The IP address from where the user that created the query logged in.
+    - name: client_ip
+      description: >
+        The IP address from where the user that created the query logged in.
+    - name: query_time.sec
+      type: float
+      description: >
+        The total time the query took, in seconds, as a floating point number.
+    - name: lock_time.sec
+      type: float
+      description: >
+        The amount of time the query waited for the lock to be available. The
+        value is in seconds, as a floating point number.
+    - name: rows_sent
+      type: long
+      description: >
+        The number of rows returned by the query.
+    - name: rows_examined
```

Comment:
I guess that this field should be a group field called `rows` with two fields inside: `sent` and `examined`